src/panels/config/core/ha-config-section-core.js

```diff
@@ -19,37 +19,6 @@ import "./ha-config-url-form";
 class HaConfigSectionCore extends LocalizeMixin(PolymerElement) {
   static get template() {
     return html`
-      <style include="iron-flex ha-style">
```

Comment:
We also don't use anything from `ha-style`?

Reply:
I checked before and after and could not spot any visual difference, so I just assumed "no" :) .